<ui version="4.0" >
 <class>GameMenuDialog</class>
 <widget class="QWidget" name="GameMenuDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>118</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Baarbarity</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item>
    <widget class="QPushButton" name="optionsButton" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Video options...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="shaderEditorButton" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Shader editor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="gameEditorButton" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Game editor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loadButton" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exitButton" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="returnButton" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Return to game</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>optionsButton</tabstop>
  <tabstop>shaderEditorButton</tabstop>
  <tabstop>gameEditorButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>returnButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
